python/importer/main.py

```diff
@@ -53,7 +54,7 @@ def main(importer: func.TimerRequest, context: func.Context):
                                 "rid of the lock by restarting the database if needed.")
                     return
                 print("Running import_day_data_from_past")
-                # import_day_data_from_past(1, cur)
+                import_day_data_from_past(1, cur)
```

Comment:
In the long run, commenting out these function calls depending on the need gets a bit tedious. Could we find a nice way to parameterize this? E.g., could we bind the "number of days since today to import from" parameter somehow to the Azure function call, perhaps with a decent default value? For development use, one could then give a different (smaller) value, or `None` if we want to skip the import part completely.